core/modules/cart/store/actions.ts

```diff
@@ -387,6 +388,20 @@ const actions: ActionTree<CartState, RootState> = {
       return diffLog
     }
   },
+  updateVariant (context, { product, configuration }) {
```

Comment:
Rename it to “configureItem” to make it coherent with “addIten”, “renoveItem” and also with “product/configure” actiona